src/concepts/config-intro.md

```diff
@@ -82,9 +107,13 @@ In the [http-kafka binding](../reference/config/bindings/binding-http-kafka.md),
 
 A corresponding `routes[].when` object with a matching `GET` method and `location` path is also required for follow-up `GET` requests to return the same response as would have been returned if the `prefer: respond-async` request header had been omitted.
 
+### Route Exit
+
+A route exists to direct messages on the stream to a desired exit point. This is the next binding needed to parse the the stream data. Bindings like [tcp](../reference/config/bindings/binding-tcp.md) are frequently used to route incoming streams to different exit points.
```

Comment:
Let's make sure it's clear we route streams, not individual messages, per related comment above.